Dockerfile.mini

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.12.4 as builder
+WORKDIR /external-dns
+COPY . .
+RUN make build
+
+FROM gcr.io/distroless/static
```

Comment:
I think we should use base instead of static because it adds support for:
- glibc
- libssl
- openssl

WDYT?

Reply:
It wouldn't do any harm here

Reply:
I also saw that we have some providers which are using crypto tls.

Reply:
AFAIK cgo is only required for crypto/tls for Darwin targets.  While using the distroless/base image instead of static may not cause harm, inclusion of those libs could make some CVEs pop up in the scans.

I'm not opposed to using `distroless/base` if these extra libs are a requirement of the binary.

Reply:
👍